c++/src/capnp/rpc.capnp

```diff
@@ -951,8 +950,8 @@ struct CapDescriptor {
     # Hopefully this is unusual.
 
     senderHosted @1 :ExportId;
-    # A capability newly exported by the sender.  This is the ID of the new capability in the
-    # sender's export table (receiver's import table).
+    # A capability exported by the sender.  This may or may not be a new ID in the sender's export
```

Comment:
This feels a little unclear to me. Maybe: "This is the ID of the capability in the sender's export table (receiver's import table). It may be a newly-allocated table entry, or an existing entry (incrementing the refcount)."